Contributing.md

```diff
@@ -0,0 +1,98 @@
+
+We have found that robust discussion before starting a new feature or large refactor has the greatest chance of success. Discussions can occur in the issue created or in Slack.
+
+
+**Note** sometimes during an active response focus is on immediate needs and those of our partners. We respectfully ask that you use the issues and `@` one of the contributors.
+
+### Whoa! I just want to fix a typo.
+
+Yes! We understand that frustration. GitHub does a great job of providing an in-browser experience for this scenario. If you don't have a GitHub account, first [create an account](https://github.com/join), then we ask you to create an issue for your suggested fix. Then navigate to the file with the minor change. In the upper right corner, next to the trash can symbol is a pencil. That icon will allow you to start the editing process. GitHub automagically will create a fork with your change and walk you through submitting it as a pull request.
+
+## Commit Messages
+
+The message should be a short summary (max. 50 chars) written in the imperative, followed by an optional, more detailed explanatory text separated from the summary by an empty line.
+
+Commit messages should follow best practices, including explaining the context of the problem and solution, including in caveats or follow up changes required. They should tell the story of the change and provide readers understanding of what led to it.
+
+If you're lost about what this even means, please see [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) for a start.
+
+
+## Pull Requests
+
+All Pull Requests should have an accompanying issue and will require review before merging.
+
+**Note During an active response, this policy can be fluid, as events on the ground change and fixes are often needed ASAP. During those times, we ask you get an explicit OK or instruction from a maintainer before creating a PR not tied to an issue.
+
+Only maintainers can merge PRs, and only they can decide whether to waive or defer a review on a PR. As a rule, this doesn't happen often, and only does during an active response or major service outage.**
+
+We ask that you follow the template for Pull Requests in the same manner as issues.
+
+We also ask that if your PR no longer merges cleanly, use `rebase master` rather than `merge master`.
+
+Include an issue reference like Closes #XXXX or Fixes #XXXX in commits that close an issue. Including references automatically closes the issue on a merge.
+
+## Review
+
+All Pull Requests require peer review before merging to master. Pull Requests should also pass all tests before consideration of merging unless there is a clear reason.
+
```

Comment:
Do we want to add a proactive statement here re-iterating the statement on line 60 about discussing large refactors prior to starting and add that it's highly likely that unsolicited/undiscussed major changes will be closed with a note to join slack and discuss further?